src/style-auro-calendar-cell.scss

```diff
@@ -55,23 +50,7 @@
   user-select: none;
 
   &.disabled {
-    color: var(--ds-color-text-secondary-inverse, $ds-color-text-secondary-inverse);
-
-    pointer-events: none;
-  }
-
-  &.selected {
-    border-color: var(--ds-color-background-darker, $ds-color-background-darker);
-
-    background-color: var(--ds-color-background-darker, $ds-color-background-darker);
-    box-shadow: 0 .1px 5px rgba(0, 0, 0, .12), 0 5px 10px rgba(0, 0, 0, .1);
-    color: var(--ds-color-background-lightest, $ds-color-background-lightest);
-
-    &:not(:hover) {
-      ::slotted([slot^="date_"]) {
-        color: var(--ds-color-text-primary-inverse, $ds-color-text-primary-inverse);
-      }
-    }
+    cursor: default !important;
```

Comment:
Are we 100% sur ethe `!important` flag is required here? It's ussually a sign that some other code could be better written.

Reply:
Yes, the `!important` flag is required here. Without it, the cursor still shows the pointer icon when the button is disabled.